### DISCUSSION TITLE
2nd Renaissance Study Group: Themes, Theories, Concepts & Questions

### DISCUSSION THREAD

**Discussion by elisapaka** [life-itself/community#1023] [Research]
A place to capture and discuss any themes, theories and concepts emerging from the [2nd Renaissance Study Group](https://github.com/orgs/life-itself/discussions/1022).

## Theories & Concepts

- **Right and left hemispheres**: "The left hemisphere's goal is to enable us to manipulate things, whereas the goal of the right hemisphere is to relate to things and understand them as a whole."
- **Opponent processing**: A psychological theory that attempts to explain 'positive' and 'negative' relationships between certain processes, how two processes 'check and balance' one another. From Wikipedia: "These responses are controlled by opponent neurons, which are neurons that have an excitatory response to some wavelengths and an inhibitory response to wavelengths in the opponent part of the spectrum."
- **Religio** (Vervaeke)

### Session 1

- 'Holding two things at once'
- Discourse as it relates to developmental stages

**Comment by farandfew** [comment]
re Holding two things at once: @rufuspollock said something intriguing that I'd like to hear more about. He said both that The Master and His Emissary was genius and potentially transformative and also that he didn't really buy much of its argument. I'd like to hear more about this.
Been wondering a bit how reception of McGilchrist's work is similar to or different from that of other people who speak in a 'scientific' context but give succour to those uncomfortable with 'scientism'. I am thinking of James Lovelock, Suzanne Simard and Rupert Sheldrake. Sure I should add some physicists too.
Those two points (about what Rufus said and about non-scien...tistic(?) scientists) are related in my mind but maybe not in reality.
